### PR TITLE
cleanup dpdk tests

### PR DESCRIPTION
**Pull request by drunkirishcoder**

### CODE REVIEW

File: framework/tests/packets.rs [hunk 1]
```diff
@@ -129,291 +123,296 @@ fn packet_from_bytes(bytes: &[u8]) -> Packet<NullHeader, EmptyMetadata> {
...
-
-    // Check Ethernet header
-    let epkt = pkt.parse_header::<MacHeader>();
-    {
-        let eth = epkt.get_header();
-        assert_eq!(eth.dst.addr, MacAddress::new(0, 0, 0, 0, 0, 1).addr);
-        assert_eq!(eth.src.addr, MacAddress::new(0, 0, 0, 0, 0, 2).addr);
-        assert_eq!(eth.etype(), Some(EtherType::IPv6));
-    }
+    dpdk_test! {{
```
Comment (drunkirishcoder) [inline framework/tests/packets.rs]: A little ugly. I don't know how to get rid of one of the `{}`.

Comment (drunkirishcoder) [reply framework/tests/packets.rs]: Let's vote, which is better?

1. `{{ }}`
2. `({ })`
3. `[{ }]`

Comment (zeeshanlakhani) [reply framework/tests/packets.rs]: 2

Comment (zeeshanlakhani) [reply framework/tests/packets.rs]: and yes, don't think you can get around block without `{}` from what i've seen typically.

**Comment by zeeshanlakhani** [comment]
@drunkirishcoder use descriptions in your PR dawg. comeon yo.

**Comment by drunkirishcoder** [comment]
My code is self-describing. I don't need descriptions!

File: framework/tests/packets.rs (hunk 1 above)
Comment (drunkirishcoder) [reply framework/tests/packets.rs]: Ok, don't need to vote anymore. Figured it out. No more doubling up.

**Pull request merged**